src/components/Tags/Tags.st.css

```diff
@@ -80,27 +80,43 @@
 .tag {
     padding: 0 value(defaultTagInnerPadding);
     max-width: value(defaultMaxWidth);
-    border-style: solid;
     margin: fallback(value(MainVerticalSpacing), value(defaultTagVerticalSpacing)) fallback(value(MainHorizontalSpacing), value(defaultTagHorizontalSpacing));
-    box-sizing: content-box;
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
```

Comment:
So actually, the guidelines we received is that it should be `content-box`.
That's how it was implemented in `wix-ui-santa`.
I'm checking to see if we can leave this like this or not, and I'll get back to you.

Reply:
I'm changing back to content box
the main reason for border box was to make the border grow inside

Reply:
Cool 👍 

Btw, I think we also we forgot to add the option to override the font: https://zeroheight.com/7sjjzhgo2/p/05a744-05-selections/t/9906bf

Sizes have different default font-sizes but they should be able to be overridden

Reply:
but i ended in a different solution

Reply:
font is already overridable one problem that i found was in that i set the height of the box to be from the lineHeight in the font definition but once user override it the lineHeight is gone so i removed the lineHeight and set min height for larger font size

Reply:
there some things that only after you try do use them you find the initial implementation was flawed

Reply:
in wix-base-ui font theme and font size is a total package you have to define them both when you override

Reply:
I think what we had was ok.
If the user changes the font, then it will change.
It will no longer be a specific size or theme.
So I think you can revert the min-height additions.

In addition, we need to add the font control in the storybook's settings panel

Reply:
![Screen Shot 2020-03-22 at 15 34 47](https://user-images.githubusercontent.com/12441962/77250773-c6ffea80-6c52-11ea-86ec-2dd22deace50.png)

this is what happening when the user overrides the font and no min-height is set
if we won't deal with it in the component then each user will need to set it itself

Reply:
Then maybe we can have a default padding, no?
I think it will work well with the outer border.